db/tracker.csv

```diff
@@ -0,0 +1,17 @@
+tracker_id,user_id,thought,feeling_id, track_date
```

Comment:
I don't think this is formatted correctly! Take a look at the other data files. I don't think we need to include the column names for each row & all columns are broken down by the commas (spaces aren't needed). I don't think this is a high priority, but if you decide to fix this later, make sure to remove the `copy` for in the  import file.

Reply:
OK going to do that now thanks for the review